tests/Badge/Model/UseCase/CreateComposerLockBadgeTest.php

```diff
@@ -96,16 +101,16 @@ public function testShouldCreateComposerLockBadge($returnCode, $expected): void
     public function testShouldCreateDefaultBadgeOnError(): void
     {
         $package = $this->createMockWithoutInvokingTheOriginalConstructor(
-            '\App\Badge\Model\Package',
+            \App\Badge\Model\Package::class,
```

Comment:
`Package::class`?

Reply:
OK, I upgraded to Docker 19.03.8

Reply:
There's already a Package name defined, it's `Packagist\Api\Result\Package`

Reply:
we can use an alias as `AppPackage`?